conans/cli/command.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, method, group, formatters=None):
             self._parser.add_argument('-o', '--output', default=default_output, choices=formatters_list,
                                       action=OnceArgument, help=self._output_help_message)
 
-    def run(self, conan_api, *args, **kwargs):
+    def run(self, *args, conan_api=None, **kwargs):
```

Comment:
Is there any invocation to this method that will not pass the ``conan_api`` argument?